File: 14_CursoProgramOrientObjects/Python/card.py
from payment import Payment
from datetime import date

class Card(Payment):
    franquise = str
    expireDate = date.today()
    cvv = str

    def __init__(self, id, franquise, expireDate, cvv):
        super.__init__(id)
        self.franquise = franquise
        self.expireDate = expireDate
        self.cvv = cvv

File: 14_CursoProgramOrientObjects/Java/Card.java
import java.util.Date;

public class Card extends Payment {
    String franquise;
    Date expireDate;
    String codigoCvv;
    public Card(Integer id, String franquise, Date expireDate, String codigoCvv){
        super(id);
        this.franquise = franquise;
        this.expireDate = expireDate;
        this.codigoCvv = codigoCvv;

        this.printDataPayment();
    }
}